bridge/client/app/project-board/project-board.component.spec.ts

```diff
@@ -156,6 +155,24 @@ describe('ProjectBoardComponent', () => {
     });
   });
 
+  it('should not show notification indicator', () => {
+    expect(fixture.nativeElement.querySelector('.notification-indicator.notification-indicator-absolute')).toBeFalsy();
+
+    component.hasUnreadLogs$ = of(true);
```

Comment:
As it is available already as test - maybe it makes sense if you test:
`should first show an notification indicator and then not if hasUnreadLogs is toggled`